javascripts/chooseplayer.js

```diff
@@ -9,16 +11,22 @@ var FoodFight = (function(choose){
     FoodFight.setUpGame(currentHero, currentEnemy);
   });
 
+  heroBtn.click(function() {
+    selected = heroBtn.filter(":checked").val();
+    console.log(selected);
+  });
+
   choose.setHero = function(){
-    var hero = FoodFight.Larry;
+    var name = userName.val()
+    var hero = new FoodFight[selected](name);
     return hero;
-    //find the thing that is chosen and set it to hero. 
+    //find the thing that is chosen and set it to hero.
   };
 
   choose.generateEnemy= function(){
     var enemy = FoodFight.player2;
```

Comment:
I believe we need to call the function that's generating the enemy here- and I think keith has that embedded in the player2 object for now- I think we need it to be enemy = foodFight.generateEnemy(); and pull lines 10 through 20 in keith's enemy.js into the main iffe scope/not underneath the enemy.player2 function.